ivadomed/loader/utils.py

```diff
@@ -46,6 +46,10 @@ def split_dataset(path_folder, center_test_lst, split_method, random_seed, train
     X_val = []
     if split_method == 'per_center':
         # make sure that subjects coming from some centers are unseen during training
+        if len(center_test_lst) == 0:
+            centers = list(df['institution_id'])
+            center_test_lst, _ = train_test_split(centers, train_size=test_frac, random_state=random_seed)
```

Comment:
Question: what happens if test_frac=0.2 but there is only 3 centers?

Reply:
There will we an error indicating that the set will be empty (test_frac should be changed), I could also add a condition so that the minimum test_frac has to be 1/n_centers, what do you think?

Reply:
Great!